modules/rest-api/pages/rest-setting-security.adoc

```diff
@@ -0,0 +1,173 @@
+= Manage Encryption Settings
+:page-topic-type: reference
+
+[abstract]
+Establish and retrieve cluster-wide settings for the use of encryption and cipher-suites.
+
+== HTTP method and URI
+
+----
+GET /settings/security
+
+POST /settings/security
+----
+
+== Description
+
+Allows settings to be established for the use of encryption.
+This includes the encryption _level_, for node-to-node encryption within the cluster;
+UI disablement over insecure connections; TLS minimum version; and a list of server-accepted cipher suites.
+
+[#curl-syntax]
+== Curl Syntax
+
+----
+curl -X GET -d hostname=<host>:<port>
```

Comment:
This request seems to be wrong. I think you meant:
```
curl -u <administrator>:<password> \
  http://<host>:<port>/settings/security
```

Reply:
Thanks.